rad/src/filters/deviation.rs

```diff
@@ -619,4 +627,40 @@ mod tests {
 
         assert_eq!(output_i128, expected);
     }
+
+    #[test]
+    fn test_filter_deviation_standard_array_of_empty_arrays() {
+        let num_witnesses = 1;
+        // Reveal value:
+        // 1D array: []
+        let input = vec![];
+        // Tally input:
+        // 2D array: [[]; num_witnesses]
+        let input_vec = vec![RadonTypes::Array(RadonArray::from(input)); num_witnesses];
```

Comment:
This is probably a different issue, but this input also panics with message "Only 2D arrays are supported"

```suggestion
        let input_vec = vec![RadonTypes::Array(RadonArray::from(vec![
            RadonTypes::Integer(RadonInteger::from(9)),
            RadonTypes::Array(RadonArray::from(vec![])),
        ])); num_witnesses];
```

Reply:
I am checking that with this PR it also works

Reply:
My bad, it should be like this:

```rust
let inner_arr = RadonTypes::Array(RadonArray::from(vec![
            RadonTypes::Integer(RadonInteger::from(9)),
            RadonTypes::Array(RadonArray::from(vec![
                RadonTypes::Integer(RadonInteger::from(9)),
            ])),
        ]));
```